src/ProjectContents/ModalNewContent.css.ts

```diff
@@ -0,0 +1,254 @@
+import { style } from "@vanilla-extract/css";
+import { s } from "../@styles";
+import { COLORS } from "../@config/colors";
+
+export const container = style([
+  s.flexColumn,
+  s.border,
+  s.scroll,
+  {
+    margin: "1rem",
+    padding: "0 2rem",
+    width: "30rem",
+    minHeight: "40rem",
+    maxHeight: "40rem",
+    backgroundColor: COLORS.WHITE,
+  },
+]);
+
+export const header = style([
+  s.flexColumn,
+  {
+    position: "sticky",
+    top: 0,
+    padding: "2rem 0",
+    backgroundColor: "inherit",
+    fontWeight: 400,
+  },
+]);
+
+export const nav = style([s.flex, { gap: "1rem", paddingTop: "2rem " }]);
+
+export const footer = style([
+  s.flex,
+  s.flexSpaceBetween,
+  {
+    flexDirection: "row-reverse",
+    position: "sticky",
+    bottom: 0,
+    padding: "1rem 0",
+    backgroundColor: "inherit",
+  },
+]);
+
+export const wrapper = style([
+  s.scroll,
+  { height: "30rem", paddingBottom: "3rem", fontSize: "0.95rem" },
+]);
+
+export const menu = style([
+  s.pointer,
+  {
+    ":hover": {
+      borderBottom: "1px solid black",
+    },
+  },
+]);
+
+export const highlightMenu = style({
+  borderBottom: "1px solid black",
+});
+
+export const headerFirstLine = style([s.flex, s.flexSpaceBetween, {}]);
+
+export const schemaNameSection = style([
+  s.flexColumn,
+  { gap: "2rem", height: "30rem;" },
+]);
+
+export const unavailableOption = style([
+  s.border,
+  {
+    padding: "1rem",
+    borderColor: "white",
+    backgroundColor: COLORS.STRAWBERRY_LIGHT,
+  },
+]);
+
+export const warningMessage = style({
+  padding: "0 1rem 1rem 1rem",
+  fontSize: "0.8rem",
+  color: COLORS.RED,
+});
+
+export const schemaName = style({ color: COLORS.LAVENDAR_DARK });
+
+export const schemaNameInput = style([
+  {
+    position: "relative",
+    borderRadius: "1rem",
+    border: "1px solid black",
```

Comment:
```suggestion
    border: `1px solid ${COLORS.BLACK}`,
```

색상은 상수화된 색상을 쓰면 좋을 것 같습니다!

Reply:
넵!